source/connection-monitoring-and-pooling/tests/README.rst

```diff
@@ -36,7 +36,21 @@ Unit Test Format:
 
 All Unit Tests have some of the following fields:
 
-- ``poolOptions``: if present, connection pool options to use when creating a pool
+- ``poolOptions``: If present, connection pool options to use when creating a pool;
+  both `standard ConnectionPoolOptions <https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#connection-pool-options-1>`__
+  and the following test-specific options are allowed:
+
+  - ``backgroundThreadDelayMS``: An artificial delay before starting observable activities in a
+    `Background Thread <https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#background-thread>`__.
+    The Test Runner / Connection Pool MUST try to start counting time towards this delay
```

Comment:
Can we change this to start counting when the pool transitions from paused to ready, since the thread won't be running when the pool is paused anyways? (only make this change assuming we stick with a delay here instead of an on/off flag per my other comment)

Reply:
I went the path proposed in the other comment.